build/windows/nanoserver/Dockerfile

```diff
@@ -1,8 +1,10 @@
-FROM microsoft/nanoserver
+ARG SOURCE_TAG=latest
```

Comment:
What's the point of adding this ARG layer?

Reply:
I needed that to be able build Windows Server, version 1803 compatible version on Win 10 (docker build . --build-arg SOURCE_TAG=1803) other why it defaults to Windows Server 2016 based version.

Reply:
@seal-ss can I still use your `rebase-docker-image` tool to build a 1803 image? If so, I believe that this line is not required.

Reply:
Probably. I was just not familiar with that tool. I can test and remove ARG layer if that works.

Reply:
Yeah our build workflow is not public yet, I still have all the docs on my side. I can give you more details on Slack at https://portainer.io/slack/, ping me there

Reply:
Yes, the `rebase-docker-image` tool still should work for your COPY deployment builds.